src/css_to_emotion.re

```diff
@@ -29,14 +29,6 @@ open Longident;
 open Css_types;
 open Component_value;
 
-module Option = {
-  let get = (opt, def) =>
```

Comment:
4.8 introduced the same function

https://caml.inria.fr/pub/docs/manual-ocaml/libref/Option.html